airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -561,6 +569,39 @@ def dry_run(self) -> None:
         pod = self.build_pod_request_obj()
         print(yaml.dump(prune_dict(pod.to_dict(), mode='strict')))
 
+    def _patch_deprecated_k8s_settings(self, hook: KubernetesHook):
+        """
+        Here we read config from core Airflow config [kubernetes] section.
+        In a future release we will stop looking at this section and require users
+        to use Airflow connections to configure KPO.
+
+        When we find values there that we need to apply on the hook, we patch special
+        hook attributes here.
+        """
+
+        # default for enable_tcp_keepalive is True; patch if False
+        if conf.getboolean('kubernetes', 'enable_tcp_keepalive', fallback=None) is False:
```

Comment:
I'm curious, why do we need `fallback=None`?

Reply:
if the value is missing from the config and you don't have a fallback, it would throw an error.
as to why None, well, just a reasonable default that means "not set".  Only when it's explicitly set to False would we want to set the hook to disable TCP keepalive.

Reply:
For those following along at home, _some_ of these fallbacks had no effect (like this one) as conf falls back to the defaults first. However, some of the others have no default ever, and are necessary.